pylint_odoo/test/main.py

```diff
@@ -9,6 +9,7 @@
 
 
 EXPECTED_ERRORS = 52
+EXPECTED_ERRORS += 2  # Errors due to C8106: Wrong version format
```

Comment:
@lescobarvx why not `EXPECTED_ERRORS = 54` in one line and add the comment in the commit message?